### DISCUSSION TITLE
[Feature Request] Task Manager - Open new windows on middle-click

### DISCUSSION THREAD

**Discussion by waenhill** [lxqt/lxqt#1887] [Ideas]
_Copy pasted from there: https://github.com/lxqt/lxqt-panel/issues/1510_

I would like to summit an idea for the Task Manager widget.
Could it be possible to add an option to open a new window on the middle-click?

The option to "close on middle-click" is available; but coming form Windows, I am used to use that shortcut to open a new one instead :)

Please let me know your comment.
Thanks you in advance!

**Comment by waenhill** [comment]
I know this issue has already been closed with the comment 
`The answer is no. Opening a new window doesn't make sense for all apps, while closing a window does.`

Please bear with me, I don't want to spam but to understand, discuss and see if anyone has an alternative maybe.

Also yes I agree that it doen't make sence for all app; But most of application are allowing several instances.
If not, then the app is already handling itself and the new window, right?

It would be similar to trying opening an app several times from "quicklaunch" widget, or from the "Application menu".

I don't have a global vision, but maybe there are apps for which it could be a problem?

Once again , then you for reading this subject.

**Reply by tsujan** [reply]
> Please bear with me, I don't want to spam

Nobody thinks that you do :)

> If not, then the app is already handling itself and the new window, right?

No! We don't know which command opens a new window in each case and, even if we magically knew it for all apps, the task manager isn't a place for issuing commands.

**Comment by stefonarch** [comment]
Some applications cannot even create a second window (all messengers, thunderbird, feed reader and so on).
IMHO the place for this is tray/statusnotifier and it's already done by many apps but not by middle click.
![schermata-07-58-48](https://user-images.githubusercontent.com/10681413/102181690-603ef300-3eab-11eb-8b2c-b1607e098539.png)
